03_ScienceExamples/MagellanicClouds/ExtremelyMetalPoorStarsInTheLMC.ipynb

```diff
@@ -0,0 +1,962 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.        AND s.g_psf &lt;= 17.5
</pre><p>I don't see this listed in the selection criteria list. I might miss it, but please double check.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astro-datalab/notebooks-latest/pull/260/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>